Libraries/react-native/react-native-implementation.js

```diff
@@ -167,6 +168,14 @@ module.exports = {
     return require('VirtualizedList');
   },
   get WebView() {
+    if (!showedWebWiewDeprecation) {
+      console.warn(
+        'WebView has been extracted from react-native core and will be removed in a future release. ' +
+          'See https://github.com/react-native-community/react-native-webview for more information',
```

Comment:
Looks good!

I would consider adding a note about the new `npm` package, similar to what Navigator is doing: https://github.com/facebook/react-native/blob/ff329801edbd7bbfd52716f8e236d13cd46b04a1/Libraries/react-native/react-native-implementation.js#L357

Reply:
Fixed 😄